### DISCUSSION TITLE
Installation Part DB

### DISCUSSION THREAD

**Discussion by debusser** [Part-DB/Part-DB-server#607] [General]
Can you install Part DB also on a Ubuntu distro? I have a Partkeepr DB running from 2017. But its no longer mantained.  And i'm looking for a good alternative.

**Comment by tpazdera** [comment]
Ubuntu is Debian based so I don't see why it would not work.  Check out the docs that talk about installation on Debian and migration from Partkeepr.

https://docs.part-db.de/installation/installation_guide-debian.html
https://docs.part-db.de/partkeepr_migration.html

**Reply by debusser** [reply]
I started to try installing it. Ubuntu has a problem with the Sury repository… not trusting it as a save package source. Since i am far from being a linux expert, i keep on trying…

**Reply by jbtronics** [reply]
If you are running a recent Ubuntu, you probably won't need it anyways. The sury repo was just required, because the Debian version for which the guide was written did not shipped the PHP Version required for Part-DB.

For newer versions of Debian and Ubuntu this should not be required anymore.
